app/helpers/hostgroup_description_helper.rb

```diff
@@ -0,0 +1,14 @@
+module HostgroupDescriptionHelper
+  UI.register_hostgroup_description do
+    hostgroup_actions_provider :base_hostgroup_actions
+  end
+
+  def base_hostgroup_actions(hostgroup)
+    actions = []
+    actions << { :action => [display_link_if_authorized(_('Nest'), hash_for_nest_hostgroup_path(:id => hostgroup))], :priority => 1 }
+    actions << { :action => [display_link_if_authorized((link_to _('Create Host'), new_host_path(hostgroup_id: hostgroup.id)))], :priority => 10 }
+    actions << { :action => [display_link_if_authorized(_('Clone'), hash_for_clone_hostgroup_path(:id => hostgroup))], :priority => 10 }
+    actions << { :action => [display_delete_if_authorized(hash_for_hostgroup_path(:id => hostgroup).merge(:auth_object => hostgroup, :authorizer => authorizer), :data => { :confirm => warning_message(hostgroup) })], :priority => 100 }
```

Comment:
Let's fix the priorities:
```suggestion
    actions << { :action => [display_link_if_authorized(_('Nest'), hash_for_nest_hostgroup_path(:id => hostgroup))], :priority => 10 }
    actions << { :action => [display_link_if_authorized((link_to _('Create Host'), new_host_path(hostgroup_id: hostgroup.id)))], :priority => 20 }
    actions << { :action => [display_link_if_authorized(_('Clone'), hash_for_clone_hostgroup_path(:id => hostgroup))], :priority => 30 }
    actions << { :action => [display_delete_if_authorized(hash_for_hostgroup_path(:id => hostgroup).merge(:auth_object => hostgroup, :authorizer => authorizer), :data => { :confirm => warning_message(hostgroup) })], :priority => 40 }
```
This would allow 9 places between each item

Reply:
Thanks @ShimShtein, I fixed your notes.

Reply:
@ShimShtein can you please take a look at the changes I made?